CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ set(SOURCE_FILES
         src/shaka_scheme/system/base/Bytevector.cpp
         src/shaka_scheme/system/base/Character.cpp
         src/shaka_scheme/system/base/Vector.cpp
-        )
+        src/shaka_scheme/system/gc/init_gc.cpp)
```

Comment:
Parenthesis needs to be on the next line or else this will break future merging and pull requests and create a merge conflict.